backend/src/main/java/com/example/backend/models/Product.java

```diff
@@ -0,0 +1,61 @@
+package com.example.backend.models;
+
+import java.util.Date;
+
+public class Product {
+    private long id;
+    private String name;
+    private float price;
+    private long category;
```

Comment:
Is this a "category id"? .. if that's case, rename it please to be "categoryId".